cluster/log-dump/log-dump.sh

```diff
@@ -333,15 +367,11 @@ function dump_nodes_with_logexporter() {
     fi
   done
 
-  # Collect names of nodes which didn't run logexporter successfully.
-  # Note: This step is O(#nodes^2) as we check if each node is present in the list of succeeded nodes.
-  # Making it linear would add code complexity without much benefit (as it just takes ~1s for 5k nodes).
+  non_logexported_nodes
```

Comment:
Maybe name this function `find_non_logexported_nodes`?

Reply:
done